tauri/src/app/runner.rs

```diff
@@ -293,14 +287,6 @@ fn build_webview(
   // TODO waiting for webview window API
   // webview.set_fullscreen(fullscreen);
 
-  if has_splashscreen {
```

Comment:
if I recall correctly, you can't remove this code, it's needed to use the tauri API on the splashscreen HTML page

Reply:
I don't think a splash screen would ever need access to the Tauri api. However, that would be necessary if we wanted it to be available just in case. The old code expected a file that no longer is generated, so we'd have to refactor it somehow.

Reply:
I'm on it: https://github.com/tauri-apps/tauri/pull/1186